src/main/java/org/commcare/formplayer/application/FormSubmissionController.java

```diff
@@ -252,6 +263,44 @@ private SubmitResponseBean processFormXml(FormSubmissionContext context) throws
         return context.success();
     }
 
+    private MultiValueMap<String, Object> getMultiPartFormBody(FormSession formSession) throws IOException {
+        MultiValueMap<String, Object> body = new LinkedMultiValueMap<>();
+
+        // Add any media files associated with the form
+        Path mediaDirPath = formSession.getMediaDirectoryPath(restoreFactory.getDomain(),
+                restoreFactory.getUsername(), restoreFactory.getAsUsername(), storageFactory.getAppId());
+        File mediaFile = mediaDirPath.toFile();
+        if(mediaFile.exists()) {
+            for (File file : Objects.requireNonNull(mediaDirPath.toFile().listFiles())) {
+                String contentType = FileUtils.getContentType(file);
+                if (!StringUtils.hasText(contentType)) {
```

Comment:
how would this not be able to get the content type if we are only writing files to this directory that match our content type filtering?

Reply:
We write to the directory when either one of extension match or MIME type match is successful. So it is potentially feasible for us to write a file for which content type methods return empty or null.

Reply:
But the file utils method falls back to extension anyway right?

Reply:
It does. Although JVM mapping for extension to mime types doesn't seem to be [very reliable](https://bugs.openjdk.org/browse/JDK-8273655). So if we start supporting `.doc` with Java8, we would get `null` result from `guessContentTypeFromName`.  

It might make more sense though to only worry about these extensions if and when we start supporting them and just throw an exception at the moment for these cases ?

Reply:
I am keeping it as it as for now as mobile also seems to be using the [same default content type](https://github.com/dimagi/commcare-android/blob/master/app/src/org/commcare/utils/FormUploadUtil.java#L348). Happy to revisit if there are any objections though.